SparkPost/2web2ui
# SA-512 Fix dropped filters for events search

#### AC: All filters are properly used and displayed when loading from URL.


### What Changed
 - Added `events` and `recipients` to whitelist of filters. 
 - Added `events` and `recipients` as constants and change usage in other places to use the imported constants. 

### How To Test
 
1. Go to the Events Search page. 

1. Use recipients and event types as filters. 

1. Click the share button to grab URL.

1. Enter the URL in a new page/tab. The resulting page should correctly show all the filters.
